references/common/metrics_test.go

```diff
@@ -30,10 +30,10 @@ func TestToPercentageStr(t *testing.T) {
 	v1, v2 = 10, 100
 	assert.Equal(t, ToPercentageStr(v1, v2), "10%")
 	v1, v2 = 10, 0
-	assert.Equal(t, ToPercentageStr(v1, v2), NA)
+	assert.Equal(t, ToPercentageStr(v1, v2), "N/A")
 }
 
-func TestGatherPodMX(t *testing.T) {
+func TestGetPodMetricsLR(t *testing.T) {
```

Comment:
MetricsLR is a bit confusing. Does LR means "limits & requests"? If so, I would suggest GetPodMetricsLR -> GetPodResource

Reply:
OK, I will fix it.